StreamChatClient_v3/Controllers/ChannelController.swift

```diff
@@ -12,18 +12,50 @@ extension Client {
     /// - Parameter options: Query options (See `QueryOptions`)
     /// - Returns: A new instance of `ChannelController`.
     ///
-    public func channelController(for channelId: ChannelId, options: QueryOptions = .all) -> ChannelControllerGeneric<ExtraData> {
-        .init(channelQuery: .init(channelId: channelId, options: options), client: self)
+    public func channelController(for cid: ChannelId, options: QueryOptions = .all) -> ChannelControllerGeneric<ExtraData> {
+        .init(channelQuery: .init(cid: cid, options: options), client: self)
     }
     
-    /// Creates a new `ChannelController` for the channel with the provided id.
+    /// Creates a new `ChannelController` for the channel with the provided channel query.
     ///
     /// - Parameter channelQuery: The ChannelQuery this controller represents
     /// - Returns: A new instance of `ChannelController`.
     ///
     public func channelController(for channelQuery: ChannelQuery<ExtraData>) -> ChannelControllerGeneric<ExtraData> {
         .init(channelQuery: channelQuery, client: self)
     }
+
+    /// Creates a new `ChannelController` that will create new channel.
+    ///
+    /// - Parameters:
+    ///   - id: The id for the new channel.
+    ///   - type: Type for the new channel.
+    ///   - name: Namr for the new channel.
+    ///   - imageURL: URL for an image for new channel.
+    ///   - team: Team for new channel.
+    ///   - members: IDs for the new channel members.
+    ///   - invites: IDs for the new channel invitees.
+    ///   - extraData: Extra data for the new channel.
+    /// - Returns: A new instance of `ChannelController`.
+    public func channelControllerCreatesChannel(
```

Comment:
The name looks really weird 🙈
1. If you want to use `Client` to create a channel, why didn't name it `createChannel`?
2. If you want to use `ChannelController` to create a channel, then this method should be inside `ChannelController`.

Reply:
Yep, the naming here can be improved. Ideally, it should be just another overload. What about:
```swift
public func channelController(
        createChannelWithId id: String,
        type: ChannelType,
        team: String? = nil,
        members: Set<UserId> = [],
        invites: Set<UserId> = [],
        extraData: ExtraData.Channel
    ) -> ChannelControllerGeneric<ExtraData> 
```